packages/interactivity/src/hooks.tsx

```diff
@@ -114,6 +114,7 @@ interface DirectivesProps {
 
 // Main context.
 const context = createContext< any >( { client: {}, server: {} } );
+context.displayName = 'InteractivityContext';
```

Comment:
This is a Preact context, not a React one, so I believe it's not needed in this PR.